### PR TITLE
First Hometasks

### PR DESCRIPTION
**Pull request by DmitryHontar**
Functions:
Reformate date to yyyy-mm-dd;
Convertion celsium to farenheit and vice versa;
Years with Saturdays 1 Jan;
How many days from now to new year?
Draw a table

### CODE REVIEW

File: convertTemperature.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+function convertFToC(f) {
+  c=(5*(f-32))/9;
+  return (c);
+}
+
+function convertCToF(c) {
+   f=(c*9/5)+32;
```
Comment (ivanbohdanov) [inline convertTemperature.js]: return f=(c*9/5)+32;

File: calculateSaturdays.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+function calculateSaturdays(a, b) {
+    var years = [];
+    for (i=a; i<=b; i++){
```
Comment (ivanbohdanov) [inline calculateSaturdays.js]: var i = a;

File: calculateSaturdays.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+        day = new Date(i, 0, 1).getDay();
+        if(day==6) {
```
Comment (ivanbohdanov) [inline calculateSaturdays.js]: === over ==

File: calculateDaysToNY.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+function calculateDaysToNY() {
+    var now = new Date();
+    var endOfAYear = new Date().setMonth(11, 31)
+    var daysToNY = (endOfAYear - now)/(24*60*60*1000);
+    daysToNY=Math.floor(daysToNY);
```
Comment (ivanbohdanov) [inline calculateDaysToNY.js]: return Math.floor(daysToNY);

File: convertTemperature.js (hunk 1 above)
Comment (DmitryHontar) [reply convertTemperature.js]: исправил

File: calculateSaturdays.js (hunk 2 above)
Comment (DmitryHontar) [reply calculateSaturdays.js]: исправил

File: calculateSaturdays.js (hunk 3 above)
Comment (DmitryHontar) [reply calculateSaturdays.js]: исправил

File: calculateDaysToNY.js (hunk 4 above)
Comment (DmitryHontar) [reply calculateDaysToNY.js]: исправил

File: calculateSaturdays.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+function calculateSaturdays(a, b) {
+    var years = [];
+    for (var i=a; i<=b; i++){
+        day = new Date(i, 0, 1).getDay();
```
Comment (ivanbohdanov) [inline calculateSaturdays.js]: переменная day не обьявлена

**Pull request merged**